tests/integrationv2/test_version_negotiation.py

```diff
@@ -9,13 +9,34 @@
     to_bytes, get_expected_gnutls_version
 
 
-@pytest.mark.uncollect_if(func=invalid_test_parameters)
+def test_nothing():
+    """
```

Comment:
I agree with the concern. However, some tests only test TLS 1.3, so they need to be disabled for 1.3 incompatible libcryptos. In this case, we sort of have to have to pass the test if there are no tests, since it's expected for there to be no tests.

For now, checking the logs is probably the best way to make sure the correct tests are being passed.

At some point, maybe changing all of the invalid_test_parameters passed to uncollect_if could be changed from uncollect_if to pytest.skip, and that we the skipped ones could at least be given a reason as to why they're being skipped in the logs. Also, I think all skipped tests doesn't count as no tests, unlike uncollect_if.

Reply:
Maybe an issue for pytest.skip refactor?